lib/spack/llnl/util/tty/log.py

```diff
@@ -258,7 +259,10 @@ def __init__(self, stream):
         self.stream = stream
 
     def write(self, data):
-        self.stream.write(data)
+        try:
+            self.stream.write(data)
+        except UnicodeEncodeError:
+            self.stream.write(str(codecs.encode(data, 'utf-8')))
```

Comment:
Was  this commited by accident?

Reply:
Sorry, yes this was an accident, I'll remove it. I'll mark this PR as draft until everything is clear.